example/opencv_demo.cc

```diff
@@ -133,10 +133,10 @@ int main(int argc, char *argv[])
         cvtColor(frame, gray, COLOR_BGR2GRAY);
 
         // Make an image_u8_t header for the Mat data
-        image_u8_t im = { .width = gray.cols,
-            .height = gray.rows,
-            .stride = gray.cols,
-            .buf = gray.data
+        image_u8_t im = { gray.cols,
+            gray.rows,
+            gray.cols,
+            gray.data
```

Comment:
Why is this change required?

Reply:
The direct initialization is a relatively new language feature that wasn't supported on older compilers. I've changed it, but it may not work on some older compilers.